salt/base/motd.sls

```diff
@@ -0,0 +1,42 @@
+/etc/update-motd.d/10-help-text:
+  file.managed:
+    - mode: 0644
```

Comment:
Note, you can run `tox` in the root directory to run the linter, it's picking up that these should be wrapped in double quotes:

```
[207] File modes should always be encapsulated in quotation marks
./salt/base/motd.sls:3
    - mode: 0644
    ```